src/main/cpp/render/uniform_buffer_store.cpp

```diff
@@ -69,7 +75,7 @@ namespace nova {
 
     void uniform_buffer_store::update_per_frame_uniforms() {
         // TODO: Fill in the per frame uniforms
-        per_frame_uniforms_buffer.send_data(per_frame_uniform_variables);
+        //per_frame_uniforms_buffer.send_data(per_frame_uniform_variables);
```

Comment:
the per frame uniforms don't get any data set and they do produce an Invalid operation or Value down the line somewhere. until we actually need it, it just complicates things as opengl stuff has a tendency to explode in your face